scrapscript.py

```diff
@@ -706,6 +712,8 @@ def eval_list(env: Env, exp: Object) -> typing.List[Object]:
     BinopKind.GREATER: lambda env, x, y: Int(eval_int(env, x) > eval_int(env, y)),
     BinopKind.LESS_EQUAL: lambda env, x, y: Int(eval_int(env, x) <= eval_int(env, y)),
     BinopKind.GREATER_EQUAL: lambda env, x, y: Int(eval_int(env, x) >= eval_int(env, y)),
+    BinopKind.BOOL_AND: lambda env, x, y: Int(eval_int(env, x) and eval_int(env, y)),
+    BinopKind.BOOL_OR: lambda env, x, y: Int(eval_int(env, x) or eval_int(env, y)),
```

Comment:
If possible, we should add tests for eager evaluation vs lazy evaluation (you you have lazy right now)

Reply:
Donezo